assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/index.test.js

```diff
@@ -563,6 +565,180 @@ describe( 'AudienceSelectionPanel', () => {
+			).toBeInTheDocument();
+			document
+				.querySelectorAll(
+					'.googlesitekit-audience-selection-panel__audience-creation-notice-audience .googlesitekit-audience-selection-panel__audience-creation-notice-audience-details h3'
+				)
+				?.forEach( ( element, index ) => {
+					expect( element ).toHaveTextContent(
+						index === 0 ? 'New visitors' : 'Returning visitors'
+					);
+				} );
+		} );
+
+		it( 'should display an audience creation notice for a single audience', async () => {
+			const mixedConfiguredAudiences = [
+				'properties/12345/audiences/1',
+				'properties/12345/audiences/2',
+				'properties/12345/audiences/3', // New visitors Site Kit audience.
+			];
+
+			const mixedSiteKitReportOptions = {
+				...reportOptions,
+				dimensionFilters: {
+					audienceResourceName: availableAudiences
+						.filter( ( { name } ) =>
+							mixedConfiguredAudiences.includes( name )
+						)
+						.map( ( { name } ) => name ),
+				},
+			};
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.setAvailableAudiences(
+					availableAudiences.filter( ( { name } ) =>
+						mixedConfiguredAudiences.includes( name )
+					)
+				);
+
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.setConfiguredAudiences( mixedConfiguredAudiences );
+
+			registry
+				.dispatch( CORE_UI )
+				.setValue( AUDIENCE_CREATION_SUCCESS_NOTICE_SLUG, true );
+
+			provideAnalytics4MockReport( registry, mixedSiteKitReportOptions );
+
+			const { getByText, waitForRegistry } = render(
+				<AudienceSelectionPanel />,
+				{
+					registry,
+				}
+			);
+
+			await waitForRegistry();
+
+			expect(
+				getByText( /Visitor group created successfully/i )
+			).toBeInTheDocument();
+			expect(
+				getByText( /Create groups suggested by Site Kit/i )
+			).toBeInTheDocument();
+			expect(
+				document.querySelector(
+					'.googlesitekit-audience-selection-panel__audience-creation-notice-audience  .googlesitekit-audience-selection-panel__audience-creation-notice-audience-details h3'
+				)
+			).toHaveTextContent( 'Returning visitors' );
+			// New visitors should now be showed in the list of configured audiences checkboxes.
+			expect(
+				document.querySelectorAll(
+					'.googlesitekit-selection-panel-item .mdc-checkbox__content label'
+				)[ 2 ]
+			).toHaveTextContent( 'New visitors' );
+		} );
+
+		it( 'should display an audience creation success notice when both audiences are created', async () => {
+			const mixedConfiguredAudiences = [
+				'properties/12345/audiences/1',
+				'properties/12345/audiences/2',
+				'properties/12345/audiences/3', // New visitors Site Kit audience.
+				'properties/12345/audiences/4', // Returning visitors Site Kit audience.
+			];
+
+			const mixedSiteKitReportOptions = {
+				...reportOptions,
+				dimensionFilters: {
+					audienceResourceName: availableAudiences
+						.filter( ( { name } ) =>
+							mixedConfiguredAudiences.includes( name )
+						)
+						.map( ( { name } ) => name ),
```

Comment:
Another marginal simplification which I'll apply:

```suggestion
					audienceResourceName: mixedConfiguredAudiences,
```